tests/test_task_details.py

```diff
@@ -0,0 +1,27 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from pages.home import HomePage
+
+
+class TestTaskDetails():
+
+    def test_that_person_can_view_task_details(self, base_url, selenium):
```

Comment:
Oops, I just realized that there is one thing missing here. Because this test does not change any data it can be marked as `nondestructive`. If we do not do that then we won't be able to run it against production. To mark it as `nondestructive`, you simply add the pytest nondestructive mark. See https://github.com/mozilla/oneanddone-tests/blob/master/tests/test_homepage.py#L12 for an example.

Please let me know once that is done, and my apologies for missing that the first time around.